resources/js/download.js

```diff
@@ -0,0 +1,105 @@
+/*
+ * activate modal with login form
+ * or logout user 
+ */
+function showModal() {
+    document.getElementById("download-modal").style.display = "block";
+    downloadXmlFiles();
+}
+
+function downloadXmlFiles() {
+    const xhr = new XMLHttpRequest();
+    xhr.open("GET", "/data/list", true);
```

Comment:
I would turn this into a relative link:
```suggestion
    xhr.open("GET", "data/list", true);
```